doc/guide.md

```diff
@@ -3128,6 +3130,46 @@ Examples:
 See [crontab man page](https://www.unix.com/man-page/linux/5/crontab/)
 for more information.
 
+### Timer management
+
+Okapi 4.8.0 provides service calls managing the time configuration for
+routing entries that provides the `_timer` interface. This allows
+inspecting the timer entries and modifying them - including disabling
+a timer.
+
+These services all have the prefix
+`/_/proxy/tenants/{tenant}/timers`.
+Consult the RAML for details.
+
+As an example of using timers, consider the okapi-test-module. It has
+a routing entry with timers. See the template
+[module descriptor](../okapi-test-module/descriptors/ModuleDescriptorTimer-template.json).
+When okapi-test-module is "installed", the target directory should have a "final"
+module descriptor `ModuleDescriptorTimer.json`.
+
+Start okapi in a separate teminal and then enable it:
+
+```
+cd okapi-test-module/target
+curl -d'{"id":"testlib"}' http://localhost:9130/_/proxy/tenants
+curl -d@ModuleDescriptorTimer.json http://localhost:9130/_/proxy/modules
+curl -d'[{"id":"test-timer","action":"enable"}]' \
+   http://localhost:9130/_/proxy/tenants/testlib/install?deploy=true
+```
+
+The module should now be running. It has 3 timers. List them with:
+
+```
+curl http://localhost:9130/_/proxy/tenants/testlib/timers
+```
+
+The first entry fires a timer every 20 seconds. Disable it with:
+
+```
+curl -XPATCH -d'{"id":"0_test-timer","routingEntry":{"delay":"0"}}' \
+    http://localhost:9130/_/proxy/tenants/testlib/timers
```

Comment:
Would it be possible to simplify the `PATCH` to use id? For example:
``` 
curl -XPATCH -d'{"delay":"0"}' \
    http://localhost:9130/_/proxy/tenants/testlib/timers/0_test-timer
```

Reply:
But the body would  not be a TimerDescriptor.json, then.. That would be a RoutingEntry.json.. Is that what you think is better?

Reply:
I can make this call as a option. It's a different path.. It's not really in harmondy with the GET which retrieves a TimerDescriptor.json..  So I'll keep the existing call and make this one - as another way.